po/vi.po

```diff
@@ -7097,21 +7101,24 @@ msgstr ""
 
 #: src/welcome-day-3.md
 msgid "Welcome to Day 3"
-msgstr ""
+msgstr "Chào mừng tới Ngày 3"
 
 #: src/welcome-day-3.md
 msgid "Today, we will cover:"
-msgstr ""
+msgstr "Chủ đề chúng ta sẽ học là:"
 
 #: src/welcome-day-3.md
 msgid ""
 "Memory management, lifetimes, and the borrow checker: how Rust ensures "
 "memory safety."
 msgstr ""
+"Quản lý bộ nhớ, vòng đời (lifetimes), và trình kiểm tra mượn (the borrow "
```

Comment:
quá trình kiểm mượn should be better than trình kiểm tra mượn. Sounds a bit rough with trình kiểm tra mượn